test/src/test/java/jenkins/model/JenkinsSystemReadAndManagePermissionTest.java

```diff
@@ -0,0 +1,65 @@
+package jenkins.model;
+
+import com.gargoylesoftware.htmlunit.html.HtmlForm;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.MockAuthorizationStrategy;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class JenkinsSystemReadAndManagePermissionTest {
+
+    private static final String SYSTEM_READER = "systemReader";
```

Comment:
```suggestion
    private static final String SYSTEM_READER_AND_MANAGER = "systemReaderAndManager";
```

Plus corresponding changes.